docs/tutorials/multi-product-formula.ipynb

```diff
@@ -0,0 +1,2620 @@
+    "    initial_state = Statevector(initial_state_circ).data\n",
+    "    time_evolved_state = exp_H @ initial_state\n",
+    "\n",
+    "    exact_obs = (\n",
+    "        time_evolved_state.conj()\n",
+    "        @ observable.to_matrix()\n",
+    "        @ time_evolved_state\n",
+    "    ).real\n",
+    "    exact_expvals.append(exact_obs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "949130d1",
+   "metadata": {},
+   "source": [
+    "#### Static MPF coefficients\n",
+    "\n",
+    "_Static_ MPFs are those where the $x_j$ values do **NOT** depend on the evolution time, $t$. Let's consider the order $\\chi = 1$ PF with $k_j$ Trotter steps, this can be written as:\n",
+    "\n",
+    "$$ S_1^{k_j}\\left( \\frac{t}{k_j} \\right)=e^{-i H t}+ \\sum_{n=1}^{\\infty} A_n \\frac{t^{n+1}}{k_j^n}  $$\n",
+    "\n",
+    "where $A_n$ are matrices which depend on the commutators of $F_a$ terms in the decomposition of the Hamiltonian. It is important to note that $A_n$ themselves are independent of time and the number of Trotter steps $k_j$. Therefore, it is possible to cancel out lower-order error terms contributing to $\\mu(t)$ with a careful choice of the weights $x_j$ of the linear combination. To cancel the Trotter error for the first $l-1$ terms (these will give the largest contributions as they correspond to the lower number of Trotter steps) in the expression for $\\mu(t)$, the coefficients $x_j$ must satisfy the following equations:\n",
+    "\n",
+    "$$ \\sum_{j=1}^l x_j = 1 $$\n",
+    "$$ \\sum_{j=1}^{l-1} \\frac{x_j}{k_j^{n}} = 0 $$\n",
+    "\n",
+    "with $n=0, ... l-2$. The first equation guarantees that there is no bias in the constructed state $\\mu(t)$, while the second equation ensures the cancellation of the Trotter errors. For higher-order PF, the second equation becomes $ \\sum_{j=1}^{l-1} \\frac{x_j}{k_j^{\\eta}} = 0 $ where $\\eta = \\chi + 2n$ for symmetric PFs and $\\eta = \\chi + n$ otherwise, with $n=0, ..., l-2$. The resulting error  (Refs. [\\[1\\]](#Reference1),[\\[2\\]](#Reference2)) is then\n",
+    "\n",
+    "$$ \\epsilon = \\mathcal{O} \\left( \\frac{t^{l+1}}{k_1^l} \\right).$$\n",
+    "\n",
+    "\n",
+    "Determining the static MPF coefficients for a given set of $k_j$ values amounts to solving the linear system of equations defined by the two equations above for the variables $x_j$: $Ax=b$. Where $x$ are our coefficients of interest, $A$ is a matrix which depends on $k_j$ and the type of PF we use ($S$), and $b$ is a vector of constraints. Specifically:\n",
+    "\n",
+    "$$A_{0,j} = 1 $$\n",
+    "$$A_{i>0,j} = k_{j}^{-(\\chi + s(i-1))}$$\n",
+    "$$b_0 = 1$$\n",
+    "$$b_{i>0} = 0 $$\n",
+    "\n",
+    "\n",
+    "where $\\chi$ is the ``order``, $s$ is $2$ if ``symmetric`` is ``True`` and $1$ otherhwise, $k_{j}$ are the ``trotter_steps``, and $x$ are the variables to solve for. The indices $i$ and $j$ start at $0$. We can also visualize this in matrix form\n",
```

Comment:
```suggestion
    "where $\\chi$ is the ``order``, $s$ is $2$ if ``symmetric`` is ``True`` and $1$ otherwise, $k_{j}$ are the ``trotter_steps``, and $x$ are the variables to solve for. The indices $i$ and $j$ start at $0$. We can also visualize this in matrix form:\n",
```